### DISCUSSION TITLE
drawArc Function with float angles? Workaround?

### DISCUSSION THREAD

**Discussion by HaenZ33** [Bodmer/TFT_eSPI#3051] [General]
Scenario:
I want to draw a parcial arc but I need it to start and end between full angle values. drawArc() only allows ints as input. I tested the wideline function, but it automatically gives me rounded edges which cant be controlled.

Is there a smooth workaround to get something like a drawArc but with int angles?
The option to turn off rounded edges on widelines would help. I could also draw the shape of a parcial arc with two triangles, but it wont give me the bent edges on the in- and outside if the arc gets bigger (so wouldnt the wideline function).

Any idea anyone?
Thx!

**Comment by Bodmer** [comment]
There is no floating point angle equivalent of the drawArc function in the library. You could however copy and adapt the function.
